docs/tutorial/twitter/tutorial-part-3.md

```diff
@@ -8,19 +8,39 @@ Pre-Evaluation
 
 First, we need to specify what's the data we need to transfer. For that, we need to take a look what tasks our application can handle:
 
-1.  Show the public identica timeline.
+1.  Show the public timeline.
 2.  Post a message.
 
-So it's clear that we need to fetch the public timeline (that's how it is called by identica), and we need to post a message to identica. It's time to take a look at the [identica API](http://status.net/wiki/Twitter-compatible_API) so that we know what we need to do to communicate with the service. But keep in mind that we are still on a website so we can't just send some `POST` or `GET` requests due to cross-site scripting restrictions. The one thing we can and should do is take advantage of JSONP. If you have never heard of JSONP, take some time to read the [article on ajaxian](http://ajaxian.com/archives/jsonp-json-with-padding) to get further details. In this tutorial, we won't use the service itself due to API changes and availability of the service. Instead we use a simple JavaScript file which will fake the JSON-P call.
+So it's clear that we need to fetch the public timeline, and we need to post a message. It's time to take a look at the api so that we know what we need to do to communicate with the service. But keep in mind that we are still on a website so we can't just send some `POST` or `GET` requests due to cross-site scripting restrictions. The one thing we can and should do is take advantage of JSONP. If you have never heard of JSONP, take some time to read the [article on ajaxian](http://ajaxian.com/archives/jsonp-json-with-padding) to get further details. In this tutorial, we won't use the service itself. Instead we use a simple JavaScript file which will fake the JSON-P call.
```

Comment:
I wonder if JSONP is appropriate in these days of CORS? No change necessary, but I expect that most people would do this with POST or GET to a CORS-enabled site instead of JSONP.